File: lib/models/tv_program.dart
class TvProgram {

  TvProgram({
    required this.scheduledTime,
    required this.programTitle,
  });
  String scheduledTime;
  String programTitle;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['scheduledTime'] = scheduledTime;
    map['programTitle'] = programTitle;
    return map;
  }

  @override
  String toString() => 'TvProgram{scheduledTime: $scheduledTime, programTitle: $programTitle}';
}

File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/configuration/TvProgram.kt
package uz.udevs.udevs_video_player.models.configuration

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class TvProgram(
    @SerializedName("scheduledTime")
    val scheduledTime: String,
    @SerializedName("programTitle")
    val programTitle: String,
) : Serializable